### ISSUE TITLE
Pause, Suspend-to-RAM, Wakeup, Resume causes errors

### ISSUE THREAD

**Issue by corcuma** [hashcat/hashcat#1791]
Low Priority issue here, so just FYI:

Pause hashcat, suspend windows to ram (standby), wake up the machine.
Several Erorrs are issued, and after resume gpu utilization and temp is gone:

```
nvmlDeviceGetTemperature(): Unknown Error

nvmlDeviceGetUtilizationRates(): GPU access blocked by the operating system

NvAPI_GPU_GetPerfPoliciesInfo(): NVAPI_ERROR

NvAPI_GPU_GetPerfPoliciesStatus(): NVAPI_ERROR



Hardware.Mon.#1..: Fan:  0% Core:1949MHz Mem:3802MHz Bus:16
```

**Comment by jsteube** [comment]
As it says: NVAPI_ERROR is an error inside nvidias NVAPI library. Thanks for reporting, but not a hashcat problem.
